fecgov/fec-cms
# 5964 Update Social Icons

## Summary

- Resolves #5964 

Updating the social media icons in the page footer and on the commissioner profile pages

### Required reviewers

1-2?

## Impacted areas of the application

Everywhere there is a YouTube or Twitter/X icon

## Screenshots

Before/after - footer
<img width="538" alt="image" src="https://github.com/fecgov/fec-cms/assets/26720877/2620948c-b777-4a22-80e2-1ee60662022a">

Before/after - commissioner profile pages
<img width="420" alt="image" src="https://github.com/fecgov/fec-cms/assets/26720877/8eec1c72-7bdc-429f-ab1a-932d26a7b999">

## Related PRs

None

## How to test
- pull the branch
- `npm run build`
- `./manage.py runserver`
- check any page's footer
   - the Twitter logo should be the X
   - the X icon should link to x.com instead of twitter.com… except you'll have to check your browser's status bar because x.com still redirects to twitter.com
   - the YouTube icon should be updated
- check any commissioner's profile (if they have a link to X/Twitter)
   - the Twitter logo should be the X
   - the X icon should link to x.com instead of twitter.com… except you'll have to check your browser's status bar because x.com still redirects to twitter.com
- **ARE THERE OTHER PLACES WE'RE USING THE YOUTUBE OR TWITTER ICONS?**
